cmd/snap-repair/runner.go

```diff
@@ -64,22 +76,139 @@ func (r *Repair) SetStatus(status RepairStatus) {
 	r.run.SaveState()
 }
 
+// nextCounter finds the next counter prefix to use for the output and status
+// files of a repair run
+func nextCounter(r *Repair) int {
+	counter := 0
+
+	// format: "r99.000001.2006-01-02T150405.output"
```

Comment:
It's not clear why we're using that complex format. Why isn't it simply r99.output which is overwritten atomically after the process finishes?  Below it looks like every single run will create a new file, which doesn't couple well with the fact we can retry a repair forever and create a new output file every time.

Reply:
Indeed, I simplified this now and added the error tracker